llm/llama-3/README.md

```diff
@@ -0,0 +1,340 @@
+<!-- $REMOVE -->
+# Scale Serving Llama-3 on Any Cloud or Kubernetes with SkyPilot
+<!-- $END_REMOVE -->
+<!-- $UNCOMMENT# Llama-3: Open LLM from Meta -->
+
+[Llama-3](https://github.com/meta-llama/llama3) is the latest top open-source models. It has been released with a license that authorizes commercial use. You can deploy a private Llama-3 chatbot with SkyPilot in your own cloud with just one simple command.
+
+* [Llama-3 release](https://github.com/meta-llama/llama3)
+* [Llama-3 blog](https://ai.meta.com/blog/meta-llama-3/)
+
+
+<p align="center">
+<img src="https://imgur.com/zPpY2Bg.gif" alt="Gradio UI serving Llama-3" style="width: 80%;">
+</p>
+
+
+## Why use SkyPilot to deploy over commercial hosted solutions?
+
+* No lock-in: run on any supported cloud - AWS, Azure, GCP, Lambda Cloud, IBM, Samsung, OCI
+* Everything stays in your cloud account (your VMs & buckets)
+* No one else sees your chat history
+* Pay absolute minimum — no managed solution markups
+* Freely choose your own model size, GPU type, number of GPUs, etc, based on scale and budget.
+
+…and you get all of this with 1 click — let SkyPilot automate the infra.
+
+
+## Prerequisites
+
+- Go to the [HuggingFace model page](https://huggingface.co/meta-llama/Meta-Llama-3-70B-Instruct) and request access to the model `meta-llama/Meta-Llama-3-70B-Instruct`.
+- Check that you have installed SkyPilot ([docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)).
+- Check that `sky check` shows clouds or Kubernetes are enabled.
+
+## SkyPilot YAML
+
+<details>
+<summary>Click to see the full recipe YAML</summary>
+
+```yaml
```

Comment:
Seems like some differences from the actual YAML content.

Reply:
Good catch! Updated. Thanks!